uint64 ts
uint64 bn

# 油门踏板量
float32 tr
# 刹车踏板量
float32 bk
# 方向盘转向值
float32 st
# 档位-D N R P
string g
# 控制模式--3自动驾驶模式，5远程控制模式
uint8 m
# zumouse添加
uint8 autoClean
# 底盘转向模式-0传统模式，1异向模式， 2同向模式
uint8 stVehicleMod
